docker-compose.yml

```diff
@@ -0,0 +1,19 @@
+version: '3'
+
+services:
+  app_dev:
+    build:
+      context: ./
+      dockerfile: DockerfileDev
+    ports:
+      - "8001:8001"
+    volumes:
+      - ./:/app
+      - env:/app/env
+      - data:/app/data
+      - node_modules:/app/frontend/node_modules
```

Comment:
I'm not sure we need `node_modules` here, as this is strictly frontend project directory. Am I missing something here?

Reply:
We mount the complete `app` directory to the container in [L11](https://github.com/ezaquarii/vpn-at-home/pull/28#discussion-diff-244535621R11). 
I added the three following lines to prevent ephemeral data from poluting the host. See [here](https://stackoverflow.com/questions/29181032/add-a-volume-to-docker-but-exclude-a-sub-folder).

Reply:
Thanks for the explanation.